### PR TITLE
Remove old upgrade generators. Fixes #1967

### LINKED ISSUE #1967: Get rid of upgrade 6.0.0 generator
https://github.com/projecthydra/sufia/blob/master/lib/generators/sufia/upgrade600_generator.rb

Can we merge that back into the install generator?  I'm not sure we want to support a Sufia 5 -> Sufia 7 upgrade. Having an extra generator slows the install process, and adds unnecessary indirection that makes it complicated to understand how it works.

### PR DESCRIPTION
**Pull request by jcoyne**
@projecthydra/sufia-code-reviewers

### CODE REVIEW

File: lib/generators/sufia/templates/migrations/create_proxy_deposit_requests.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class CreateProxyDepositRequests < ActiveRecord::Migration
   def change
     create_table :proxy_deposit_requests do |t|
-      t.string :pid, null: false
+      t.string :generic_file_id, null: false
```
Comment (mjgiarlo) [inline lib/generators/sufia/templates/migrations/create_proxy_deposit_requests.rb]: Shouldn't this be `:generic_work_id`?

**Comment by mjgiarlo** [comment]
Wipe out `update_content_blocks_generator.rb` too?

File: lib/generators/sufia/templates/migrations/create_proxy_deposit_requests.rb (hunk 1 above)
Comment (jcoyne) [inline lib/generators/sufia/templates/migrations/create_proxy_deposit_requests.rb]: No, there is a file in the upgrade_700 generator that will do that.

Comment (mjgiarlo) [inline lib/generators/sufia/templates/migrations/create_proxy_deposit_requests.rb]: :ok_hand:

**Pull request merged**